AsyncDisplayKit/Layout/ASLayoutSpec.mm

```diff
@@ -81,8 +81,8 @@ - (ASLayout *)measureWithSizeRange:(ASSizeRange)constrainedSize
 
     id<ASLayoutable> finalLayoutable = [child finalLayoutable];
     if (finalLayoutable != child) {
-      // Layout options state of child needs to be copied to final layoutable
-      finalLayoutable.environmentState.layoutOptionsState = child.environmentState.layoutOptionsState;
+      // Layout options state of child needs to be copied to final layoutable, but don't override customized values.
+      ASEnvironmentStatePropagateUp(finalLayoutable, child.environmentState.layoutOptionsState);
```

Comment:
This seems like a positive change. I could imagine a final layoutable object with a custom value for a layout options property that is essential not to overwrite, but it does seem like a reasonable default behavior to inherit any properties that aren't overridden.

This does raise a question, which was also present in the prior behavior, on how a final layout of all might be able to override a property to require that it stay the default (e.g. intentionally set flexGrow = NO).  It is quite rare to use finalLayoutable however, so this seems fine for now.